openid_connect_azure_b2c.module

```diff
@@ -1,8 +1,21 @@
 <?php
 
+use Drupal\Component\Serialization\Json;
 use Drupal\openid_connect\Entity\OpenIDConnectClientEntity;
 use Drupal\openid_connect_azure_b2c\Plugin\OpenIDConnectClient\OpenIDConnectB2CClient;
 
+
+function parseToken(string $token) {
```

Comment:
Can the return type be declared e.g.
`function parseToken(string $token): array {`
If this is done then the fallback could be `return [];`

Reply:
I pulled this from the core openid_connect module, so left it untouched. Could do, though?

Reply:
Updated in 73a6c4f

Reply:
Ah ok, you have to be careful if you're overriding a parent method.

Reply:
It's not a parent, otherwise I'd have called it directly. It's part of a support view for a different part of the system, but it's protected so not easily accessed.